src/main/java/com/rapid7/jenkinspider/PostBuildScan.java

```diff
@@ -192,50 +191,53 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
         }
 
         /* Scan finished */
-        if (!ScanManagement.hasReport(appSpiderEntUrl, appSpiderEntApiKey, scanId)) {
+        if (ScanManagement.hasReport(appSpiderEntUrl, appSpiderEntApiKey, scanId).orElse(false)) {
```

Comment:
Looks like you've inverted the boolean, probably need the '!' still

Reply:
whoops, yeah I'll update that